server/src/main/java/com/scalar/db/server/service/ServerModule.java

```diff
@@ -22,7 +25,11 @@
   public ServerModule(ServerConfig config, DatabaseConfig databaseConfig) {
     this.config = config;
     storageFactory = new StorageFactory(databaseConfig);
-    transactionFactory = new TransactionFactory(databaseConfig);
+
+    // For two-phase consensus commit in Scalar DB server, set MANAGE_ACTIVE_TRANSACTIONS to false
```

Comment:
I think it would be better to explain the reason for doing this in addition to what it is doing.
```suggestion
    // For two-phase consensus commit in Scalar DB server, set MANAGE_ACTIVE_TRANSACTIONS to false because Scalar DB server takes care of active transactions management
```
Is it correct?